include/mpm_explicit.h

```diff
@@ -30,6 +30,9 @@ class MPMExplicit : public MPM {
  public:
   //! Default constructor
   MPMExplicit(std::unique_ptr<IO>&& io);
+  
+  //USL or USF
+  bool USL;
```

Comment:
Why is this declared in the public section? Can we have it as a private variable. Also our naming convention is all lower case with an underscore at the end for private variables. See: https://cb-geo.github.io/cpp-guide/#/naming/naming_rules